public/app/whiteboards/reuse/reuse.html

```diff
@@ -11,18 +11,19 @@ <h2 class="modal-title">Add asset(s)</h2>
           data-search-options-category="searchOptions.category"
           data-search-options-user="searchOptions.user"
           data-search-options-section="searchOptions.section"
-          data-search-options-type="searchOptions.type"></search>
+          data-search-options-type="searchOptions.type"
+          data-ng-if="isSearch || pinned.length || assets.length"></search>
 
         <div class="col-list-container"
           data-infinite-scroll="getAssets()"
-          data-infinite-scroll-ready="list.ready"
+          data-infinite-scroll-ready="scrollReady"
           data-infinite-scroll-distance="400"
           data-infinite-scroll-container="whiteboards-reuse-list-container">
 
-          <div data-ng-if="!isSearch">
-            <h3 data-ng-if="!isSearch">Pinned Assets</h3>
-            <ul>
-              <li class="list-inline col-xs-6 col-sm-4 col-md-3" data-ng-repeat="asset in assets | filter:{pinned_by_me_date: ''} | unique:'id'">
+          <div data-ng-if="!isSearch && (pinned.length > 0 || assets.length > 0)">
```

Comment:
@paulkerschen I believe `!isSearch && ...` is necessary. In case of `isSearch` we rely entirely
 on the `All Assets` block (far below). The 'Pinned Assets' section is strictly default page view.